src/main/java/org/littleshoot/proxy/impl/TracingAwareHttpRequestEncoder.java

```diff
@@ -23,10 +23,16 @@ protected void sanitizeHeadersBeforeEncode(HttpRequest msg, boolean isAlwaysEmpt
     HttpHeaders headers = msg.headers();
 
     if (isForwardProxyTransparent()) {
+      // W3C Trace Context
+      headers.remove(TraceUtils.HEADER_VGS_TRACEPARENT);
```

Comment:
What happens if the customer was using this outside of vgs?

Reply:
I'm not really sure, the same would have been true previously with X-B3 (alternative to W3 Trace from Jaeger).

Reply:
This is from a specific preference that needs to be set to prevent us sending any vgs-generated headers upstream.